packages/code-utils/src/opcodes.ts

```diff
@@ -162,8 +160,11 @@ const codes: OpcodeTable = {
   0xf5: "CREATE2",
   0xfa: "STATICCALL",
   0xfd: "REVERT",
-  //(we can omit 0xfe INVALID)
+  0xfe: "INVALID",
   0xff: "SELFDESTRUCT"
-};
+} as const;
+
+export type OpcodeTable = typeof codes;
 
-export = (op: number) => (codes[op] ? codes[op] : "INVALID");
+export const parseOpcode = (op: number) =>
+  codes[op in codes ? (op as keyof OpcodeTable) : 0xfe];
```

Comment:
Using `0xfe` as a default before the lookup, rather than `"INVALID"` after, is pretty confusing.  I think it would be better to do this the old way (not necessarily the same code, but at least doing the defaulting after), or, failing that, there should at least be a comment to explain this.

Reply:
Okee! Yeah I was on the fence. I'll switch it back.